src/EA.Weee.Core/Organisations/RepresentingCompanyDetailsViewModel.cs

```diff
@@ -0,0 +1,30 @@
+namespace EA.Weee.Core.Organisations
+{
+    using System.ComponentModel;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.DataStandards;
+    using EA.Weee.Core.Validation;
+
+    public class RepresentingCompanyDetailsViewModel
+    {
+        [Required]
+        [DisplayName("Producer name")]
+        [StringLength(CommonMaxFieldLengths.DefaultString)]
+        public string CompanyName { get; set; }
+
+        [DisplayName("Trading name")]
+        public string BusinessTradingName { get; set; }
+
+        public ExternalAddressData Address { get; set; } = new ExternalAddressData();
```

Comment:
@stevelee-civica  here I think we need to use the EA.Weee.Core.AddressData version as the external one has a Website address which is mandatory that I don't think is on the representing company details screen.